docs/build-redirects.js

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/*
+ * This script builds FF.com's redirects
+ */
+
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+
+import chalk from "chalk";
+import fs from "fs-extra";
+
+const dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const {
+	params: { currentVersion, ltsVersion },
+} = await fs.readJSON(path.resolve(dirname, "data", "versions.json"));
+
+try {
+	const content = `/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Map of incoming URL paths to redirect URLs
+const routes = new Map([
+	["/docs/apis", "/docs/api/${currentVersion}"],
```

Comment:
This pattern doesn't work with our new setup. I actually have a prototype more or less finished locally that works with the new hierarchy, I was planning on testing it tomorrow.